src/components/HelpMenu.tsx

```diff
@@ -25,8 +25,9 @@ const HelpMenu = ({ ...rest }: HelpMenuProps) => {
   const feedbackDisclosure = useDisclosure();
   const intl = useIntl();
   const MenuButtonRef = useRef(null);
+  const ContainerRef = useRef(null);
   return (
-    <Box {...rest}>
+    <Box {...rest} ref={ContainerRef}>
```

Comment:
Do we have 2 completely different solutions to this problem for the settings menu and the help menu?

Reply:
Yes, I believe settings MenuList is using `zIndex = {2}`. Shall I change that to use portal instead?

Reply:
Yeah I think that's reasonable - probably less fragile. Either way let's make them work the same way!

Reply:
Change made here -->  [30b2bc3](https://github.com/microbit-foundation/ml-trainer/pull/488/commits/30b2bc34922e43fbb6b5c8571b785b9b24916c36)